cdap-ui/app/features/hydratorplusplus/services/hydrator-plus-ordering-factory.js

```diff
@@ -19,7 +19,7 @@ function HydratorPlusPlusOrderingFactory() {
     let artifactMap = {
       'cdap-etl-batch': 'ETL Batch',
       'cdap-etl-realtime': 'ETL Realtime',
-      'cdap-data-pipeline': 'Data Pipeline (Beta)'
+      'cdap-data-pipeline': 'Data Pipeline'
```

Comment:
@astewes Have already added this change in this PR https://github.com/caskdata/cdap/pull/6107. This involves a little bit more and we have a JIRA for it.